Dates-and-Reminders.md

```diff
@@ -0,0 +1,24 @@
+# Dates and Reminders
+
+This document will be used to capture dates and reminders to review before each CPC meeting. This is a manual process that we hope will be replaced by automated tooling at some point in the near future.
+
+## Recurring dates
+
+- CPC Chair election
```

Comment:
This year, the CPC chair election happened ending on Aug 17. Should we choose a different date to make the official end of term? Perhaps not the same date as the Secondary Director election or is it not a big deal?